src/Legacy/Drupal7.php

```diff
@@ -43,7 +43,14 @@
 class Drupal7 {
 
   public function __call($method, $args) {
-    return call_user_func_array($method, $args);
+    // Hack: See http://goo.gl/aM5fra
```

Comment:
I wouldn't call this a "hack", this is a very clever way to do something, but I don't consider this as a hack.